packages/client/src/store/layoutStore.ts

```diff
@@ -8,14 +8,12 @@ interface IBearsStore {
   toggleSidebar(): void;
 }
 
-const getColorMode = () => {
-  return () => (localStorage.getItem("mode") as "dark" | "light") || "light";
-};
-
 export const useLayoutStore = create<IBearsStore>()((set) => ({
   openSidebar: false,
-  colorModeFunc: getColorMode(),
-  colorMode: "light",
+  colorMode:
+    typeof window !== "undefined"
+      ? (localStorage.getItem("mode") as "dark" | "light")
+      : "light",
```

Comment:
1) Мы никогда не знаем, что лежит в `localStorage`. Этот код потенциально может принести багов новым пользователям. Лучше не тайпкастить, а отдельно проверять что именно мы получили в результате вызова `localStorage.getItem("mode")`

```suggestion
// ...
let colorMode: IBearsStore["colorMode"] = "light";
const allowedModes: IBearsStore["colorMode"][] = ["light", "dark"];
const savedMode = localStorage.getItem("mode");
if (typeof window !== "undefined" && savedMode && savedMode in allowedModes) {
  colorMode = savedMode;
}
// ...
  colorMode
```

2) Лучше не хардкодить тайпкаст, а всё таки ссылаться на желаемое поле, которое бы вы хотели явно указать:

```suggestion
  colorMode:
    typeof window !== "undefined"
      ? (localStorage.getItem("mode") as IBearsStore["colorMode"])
      : "light",
```